docs/source/schemas/index.rst

```diff
@@ -133,33 +133,35 @@ characteristics, among others. The types are:
 +-------------------------------+--------------+---------------+----------------------+----------------+
 | NULLABLE\_FIXED\_LONG         | long         | byte[9]       | YES                  | YES            |
 +-------------------------------+--------------+---------------+----------------------+----------------+
-| VAR\_LONG                     | long         | byte[len]\*   | YES                  | YES            |
+| VAR\_LONG                     | long         | byte[len] \*  | YES                  | YES            |
++-------------------------------+--------------+---------------+----------------------+----------------+
+| VAR\_SIGNED\_LONG             | long         | byte[len] \*  | YES                  | YES            |
 +-------------------------------+--------------+---------------+----------------------+----------------+
-| VAR\_SIGNED\_LONG             | long         | byte[len\*]   | YES                  | YES            |
+| UUID                          | UUID         | byte[16]      | YES                  | YES            |
```

Comment:
I think we probably want something to that effect in the docs, to do good schema design I think you need to know what you're doing and why you're doing it. That table is *far* easier for me to parse now I have an understanding of how things look in byte form.